clean_project/construct_dict.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""
+Construct a JSON file used to indexing all words in a config_file.yaml
```

Comment:
construct is not a word i'd use for this task. i'd rename the file into index.py or similar, and rename the function `construct_dictionary` to `index`